From 0afd3d8f94f2b1ccc87511f6b9d37f3c9921b170 Mon Sep 17 00:00:00 2001
From: Jarno Rajahalme <jarno@isovalent.com>
Date: Fri, 27 Jan 2023 13:21:34 +0200
Subject: [PATCH 6/6] tcmalloc: Update for arm64 support

arm64 builds fail due to invalid flag output constraint being used:

  error: unknown 'asm' flag output '=@ccbe'

Update to tcmalloc master that has this fixed.

Signed-off-by: Jarno Rajahalme <jarno@isovalent.com>
---
 bazel/repository_locations.bzl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/bazel/repository_locations.bzl b/bazel/repository_locations.bzl
index 613b3c0241..252c1f1b1f 100644
--- a/bazel/repository_locations.bzl
+++ b/bazel/repository_locations.bzl
@@ -336,12 +336,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "tcmalloc",
         project_desc = "Fast, multi-threaded malloc implementation",
         project_url = "https://github.com/google/tcmalloc",
-        version = "e33c7bc60415127c104006d3301c96902f98d42a",
-        sha256 = "14a2c91b71d6719558768a79671408c9acd8284b418e80386c5888047e2c15aa",
+        version = "e417bfa95505911554d6392263f85dfe6f5d8eef",
+        sha256 = "7e2506568b222ddc1e4df4c9855e4e1e9906cb3ac4c6c7978b1d25577fd7131b",
         strip_prefix = "tcmalloc-{version}",
         urls = ["https://github.com/google/tcmalloc/archive/{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        release_date = "2022-10-24",
+        release_date = "2023-01-24",
         cpe = "N/A",
         license = "Apache-2.0",
         license_url = "https://github.com/google/tcmalloc/blob/{version}/LICENSE",
-- 
2.39.2

